### ISSUE TITLE
unexpected duplicate data-name

### ISSUE THREAD

**Issue by jfhenon** [SVG-Edit/svgedit#674]
Using last Chrome version.
1. If you paste this SVG string into the source button
````
<?xml version="1.0"?>
<svg width="226.26" height="341.57" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg" xmlns:se="http://svg-edit.googlecode.com">
 <g class="layer">
  <title>Layer 1</title>
  <g data-name="Layer 1" id="Layer_1">
    <rect height="70" id="svg_2" width="20" x="100" y="200"/>
  </g>
 </g>
</svg>
````
![image](https://user-images.githubusercontent.com/20402845/144744591-6f7eea10-da44-4dc4-b8c0-f3aa74cb7c74.png)
2.Click on "Apply Changes"
3. Reopen the source again -> **Bug**:  "data-name" attribute  should not be duplicated 
![image](https://user-images.githubusercontent.com/20402845/144744620-5200b77d-d4d1-4c88-a839-46bad2273844.png)

**Comment by jggsus88** [comment]
Can I take this issue?

**Comment by jfhenon** [comment]
Sure!

**Comment by jggsus88** [comment]
Thanks, I'm on it.

**Comment by jfhenon** [comment]
fixed with #681
